doc/modules/cross_decomposition.rst

```diff
@@ -23,19 +20,170 @@ Cross decomposition algorithms find the fundamental relations between two
 matrices (X and Y). They are latent variable approaches to modeling the
 covariance structures in these two spaces. They will try to find the
 multidimensional direction in the X space that explains the maximum
-multidimensional variance direction in the Y space. PLS-regression is
-particularly suited when the matrix of predictors has more variables than
-observations, and when there is multicollinearity among X values. By contrast,
-standard regression will fail in these cases.
+multidimensional variance direction in the Y space. In other words, PLS
+projects both `X` and `Y` into a lower-dimensional subspace such that the
+covariance between `tranformed(X)` and `transformed(Y)` is maximal.
+
+PLS draws similarities with Principal Component Regression, where the samples
+are first projected into a lower-dimensional subspace, and the targets `y`
+are predicted using `transformed(X)`. One issue with PCR is that the
+dimensionality reduction is unsupervized, and may lose some important
+variables: PCR would keep the features with the most variance, but it's
+possible that features with a small variances are relevant from predicting
+the target. In a way, PLS allows for the same kind of dimensionality
+reduction, but by taking into account the targets `y`.
 
-Classes included in this module are :class:`PLSRegression`
+Apart from CCA, the PLS estimators are particularly suited when the matrix of
+predictors has more variables than observations, and when there is
+multicollinearity among the features. By contrast, standard linear regression
```

Comment:
The fact that it fixes the multicollinearity issue while linear regression would fail unless regularization seems an authoritative argument here. Do we have a ref or example that demo this?

Reply:
When features are collinear, the covariance matrix is ill-defined and thus non-invertible. 

I don't have a ref at hand, but I'm assuming this is common knowledge (could be wrong?)

Reply:
are you referring to this https://github.com/scikit-learn/scikit-learn/pull/17095/files#diff-df97917f68917d3a110df30940d771dfR176 ? if it's a statement of PLS vs CCA rather than PLS vs linear regression. Maybe I am nitpicking here.

Reply:
It's mostly a statement about PLS vs non-regularized LR: LR is unstable when there is collinearity among features, PLS is not. However, CCA is unstable too (as described in the link you mentioned).